openedx/features/caliper_tracking/tests/expected/edx.video.closed_captions.shown.json

```diff
@@ -0,0 +1,46 @@
+{
+    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "action": "Showed",
```

Comment:
There are better actions in media event. Please look into them

Reply:
Done